OKTA-488797 Updates to OAuth and OpenID Overview (#3278)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

### Interaction Code flow

> The Interaction Code flow is an extension to the OAuth 2. and OIDC standard, and is available when using Identity Engine orgs. It requires clients to pass a client ID, as well as a Proof Key for Code Exchange (PKCE), to keep the flow secure. The user can start the request with minimal information, relying on the client to facilitate the interactions with the Identity Engine component of the Okta Authorization Server to progressively authenticate the user. See [Interaction Code grant type](/docs/concepts/interaction-code/).

Reviewer: ```suggestion
The Interaction Code flow is an extension to the OAuth 2 and OIDC standard, and is available when using Identity Engine orgs. It requires clients to pass a client ID, as well as a Proof Key for Code Exchange (PKCE), to keep the flow secure. The user can start the request with minimal information, relying on the client to facilitate the interactions with the Identity Engine component of the Okta Authorization Server to progressively authenticate the user. See [Interaction Code grant type](/docs/concepts/interaction-code/).
```

-->

For information on how to set up your application to use this flow, see [Implement the SAML 2.0 Assertion flow](/docs/guides/implement-grant-type/saml2assert/main/).

### Implicit flow

 > **Note:** The Implicit flow is a legacy flow used only for SPAs that can’t support PKCE.

 The Implicit flow is intended for applications where the confidentiality of the client secret can't be guaranteed. In this flow, the client doesn't make a request to the `/token` endpoint, but instead receives the access token directly from the `/authorize` endpoint. The client must be capable of interacting with the resource owner's user agent and also capable of receiving incoming requests (through redirection) from the authorization server.

> > **Note:** Because it is intended for less-trusted clients, the Implicit flow doesn't support refresh tokens.

Reviewer: ```suggestion
> **Note:** Because it's intended for less-trusted clients, the Implicit flow doesn't support refresh tokens.
```
